src/coreclr/src/jit/instr.cpp

```diff
@@ -648,118 +648,39 @@ void CodeGen::inst_TT(instruction ins, GenTree* tree, unsigned offs, int shfv, e
     }
 }
 
-/*****************************************************************************
- *
- *  Generate an instruction that has one operand given by a tree (which has
- *  been made addressable) and another that is a register.
- */
-
-void CodeGen::inst_TT_RV(instruction ins, GenTree* tree, regNumber reg, unsigned offs, emitAttr size, insFlags flags)
+//------------------------------------------------------------------------
+// inst_TT_RV: Generate a store of a lclVar
+//
+// Arguments:
+//    ins  - the instruction to generate
+//    tree - the lclVar node
+//    reg  - the register currently holding the value of the local
+//    size - the size attributes for the store
+//
+void CodeGen::inst_TT_RV(instruction ins, GenTree* tree, regNumber reg, emitAttr size)
```

Comment:
It would be nice to reorder parameters to be more like various `emitIns` functions - `ins, size, reg, tree`

Reply:
Makes sense - this code was only tangentially related to the EHWriteThru changes, but when reviewing my changes I realized that this code was only ever used for `GT_LCL_VAR` so it made sense to simplify it.